tests/policy_as_code_test.go

```diff
@@ -248,15 +328,44 @@ func (pc *PolicyAsCodeSuite) deleteCRandObserveInCentral(k8sPolicy *v1alpha1.Sec
 	}, time.Second*5, time.Millisecond*30, "Policy CR deletion not propogated to Central")
 }
 
+func (pc *PolicyAsCodeSuite) createCRAndObserveInCentral(policyCR *v1alpha1.SecurityPolicy) string {
+	_, err := pc.k8sClient.Create(pc.ctx, pc.toUnstructured(policyCR), metav1.CreateOptions{})
+	pc.Require().NoError(err)
+
+	var policyId string
+	pc.Require().EventuallyWithT(func(collect *assert.CollectT) {
+		resp, err := pc.policyClient.ListPolicies(pc.ctx, &v1.RawQuery{})
+		if err != nil {
+			collect.Errorf("Failed to list policies: %s", err.Error())
+		}
+		for _, p := range resp.GetPolicies() {
+			if p.GetName() == policyCR.Spec.PolicyName {
+				policyId = p.GetId()
+				break
+			}
+		}
+		assert.NotEmpty(collect, policyId)
+	}, time.Second*5, time.Millisecond*30)
+	return policyId
+}
+
 func (pc *PolicyAsCodeSuite) TearDownSuite() {
 	// TODO: Don't double delete
 	for _, policy := range pc.policies {
 		log.Infof("Deleting policy with name \"%s\"", policy.Name)
-		_, err := pc.centralClient.DeletePolicy(pc.ctx, &v1.ResourceByID{
+		_, err := pc.policyClient.DeletePolicy(pc.ctx, &v1.ResourceByID{
 			Id: policy.Id,
 		})
 		pc.Require().NoError(err)
 	}
+
+	log.Infof("Deleting notifier with name \"%s\"", pc.notifier.Name)
+	_, err := pc.notifierClient.DeleteNotifier(pc.ctx, &v1.DeleteNotifierRequest{
+		Id:    pc.notifier.Id,
+		Force: true,
+	})
+	pc.Require().NoError(err)
```

Comment:
I got a panic in this code when my local test failed early for an unrelated reason
```suggestion
       if pc.notifier != nil {
               log.Infof("Deleting notifier with name \"%s\"", pc.notifier.Name)
               _, err := pc.notifierClient.DeleteNotifier(pc.ctx, &v1.DeleteNotifierRequest{
                       Id:    pc.notifier.Id,
                       Force: true,
               })
               pc.Require().NoError(err)
       }
```